app/assets/javascripts/views/task_form.js

```diff
@@ -40,7 +40,7 @@ Fulcrum.TaskForm = Fulcrum.FormView.extend({
       },
 
       error: function(model, response) {
-        if(response.responseText === undefined) return;
+        if(!response.responseText || response.responseText == '') return;
```

Comment:
Since empty string is falsy in JS, we can remove the second part of the conditional 😄